app/views/application/_search_form.html.erb

```diff
@@ -15,7 +15,9 @@
 
   <%# Preserve other parameters, e.g. for sorting %>
   <% request.query_parameters.each do |key, value| %>
-    <% next if key == 'query' %>
+    <%# Skip query, because it would overwrite the input's value %>
+    <%# also skip page, because we want to be on page 1 when we search %>
+    <% next if key == 'query' || key == 'page' %>
```

Comment:
what do you mean with "overwrite"? i found out, that when `query` is not skipped, i get it twice like `?query=a&query=a`.

Reply:
`type` being preserved it lit! so let's keep it.